### ISSUE TITLE
Freezes at startup then crashes

### ISSUE THREAD

**Issue by bNobo** [getinsomnia/insomnia#4331]
### Expected Behavior

I expect the application won't freeze and won't crash.

### Actual Behavior

When I launch the application, it immediatly freezes, memory consumption grow up to 3 GB and then it crashes

![image](https://user-images.githubusercontent.com/11414291/147251587-b9590912-d677-4fa7-aa4c-58ed3a62e487.png)

Windows Error Reporting create a report containing:

```
Détecteur d'erreurs 1542158538696669630, type 4
Nom d’événement : APPCRASH
Réponse : Non disponible
ID de CAB : 0

Signature du problème : 
P1 : Insomnia.exe
P2 : 2021.7.2.0
P3 : 601d93db
P4 : KERNELBASE.dll
P5 : 10.0.18362.1474
P6 : 5ae7af90
P7 : e0000008
P8 : 0000000000043b19
P9 : 
P10 : 

Fichiers joints :
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER6C85.tmp.dmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER6F55.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER7050.tmp.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER705F.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER70BE.tmp.txt

Ces fichiers sont peut-être disponibles ici :
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_Insomnia.exe_9a625a8870e9e00866adc97431dea5a1af631f5_0a3520f7_11aeb773-a675-455b-9e0f-00ceaf032d1f

Symbole d’analyse : 
Nouvelle recherche de la solution : 0
ID de rapport : 00b6382f-9118-4875-bdf8-aca7800ab467
Statut du rapport : 268435456
Récipient avec hachage : 0ae7c747ca3337dcc566d9052e59b9be
GUID CAB :0
```

### Reproduction Steps

Just start the application using the taskbar.

If I remove the folder `C:\Users\bnobo\AppData\Roaming\Insomnia\responses` then the app do work again but after few minutes the problem is back and I have to delete *responses* folder again.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

I made a dump analysis using DebugDiag and what it found is:

```
In Insomnia.exe.68408.dmp the assembly instruction at KERNELBASE!RaiseException+69 in C:\Windows\System32\KERNELBASE.dll from Microsoft Corporation
This exception originated from KERNELBASE!WaitForMultipleObjectsEx. has caused an unknown exception (0xe0000008) on thread 0

Entry point | Insomnia!uv_random+11b0240
-- | --
Create time | 12/23/2021 14:34:02
Time spent in user mode | 0 Days 00:00:30.593
Time spent in kernel mode | 0 Days 00:00:03.890

  | Source
-- | --
ntdll!NtWaitForMultipleObjects+14 |  
KERNELBASE!WaitForMultipleObjectsEx+107 |  
KERNELBASE!WaitForMultipleObjects+e |  
kernel32!WerpReportFaultInternal+51b |  
kernel32!WerpReportFault+ac |  
KERNELBASE!UnhandledExceptionFilter+3b8 |  
ntdll!RtlUserThreadStart$filt$0+a2 |  
ntdll!_C_specific_handler+96 |  
ntdll!RtlpExecuteHandlerForException+f |  
ntdll!RtlDispatchException+219 |  
ntdll!RtlRaiseException+153 |  
KERNELBASE!RaiseException+69 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+25f8a2 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+2ac4bf |  
Insomnia!uv_gettimeofday+302c7 |  
Insomnia!uv_os_getpid+eaf0 |  
Insomnia!uv_random+11adddf |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+221fa4 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+240bb1 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+240b26 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+240b26 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+240b26 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+240b26 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+2408da |  
Insomnia!uv_fs_get_system_error+19c1d3 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+23f672 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+291e04 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+27a589 |  
Insomnia!uv_gettimeofday+4862f |  
Insomnia!uv_gettimeofday+48241 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+2a93f6 |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+2a8d01 |  
Insomnia!uv_gettimeofday+48c7e |  
Insomnia!node::tracing::TraceEventHelper::GetCategoryGroupEnabled+263326 |  
Insomnia!uv_timer_set_repeat+1ee626 |  
Insomnia!uv_timer_set_repeat+1f0231 |  
Insomnia!uv_timer_set_repeat+1eba13 |  
Insomnia!uv_timer_set_repeat+d4109 |  
Insomnia!uv_timer_set_repeat+d3d1c |  
Insomnia!v8::Location::GetColumnNumber+ccb64 |  
Insomnia!v8::Location::GetColumnNumber+cce8d |  
Insomnia!Ordinal0+1506 |  
Insomnia!uv_random+11b01d2 |  
kernel32!BaseThreadInitThunk+14 |  
ntdll!RtlUserThreadStart+21
```

### Insomnia Version

Version: Insomnia 2021.7.2 Release date: 12/9/2021 OS: Windows_NT x64 10.0.18363 Electron: 11.2.3 Node: 12.18.3 V8: 8.7.220.31-electron.0 Architecture: x64 node-libcurl: libcurl/7.73.0-DEV OpenSSL/1.1.1g zlib/1.2.11 brotli/1.0.9 WinIDN libssh2/1.9.0_DEV nghttp2/1.41.0

### What operating system are you using?

Windows

### Operating System Version

Windows 10 version 1909 Build 18363.1556

### Installation method

download from insomnia web site

**Comment by filfreire** [comment]
@bNobo Thank you for reporting this! The issue might have something to do with your local data, but we'd need more information to try and reproduce it.

- Does this happen if you remove/rename the `Insomnia` data folder (e.g. `C:\Users\bnobo\AppData\Roaming\Insomnia`) and start with a clean one?

- Or in other words, are you able to reproduce it on a clean install, or only after importing your work?

- Do you have anything out of the ordinary on your collections/design documents? Is there anything else you can share that might help reproduce the issue?

I've tried reproducing this on different Windows 10 installations but I don't see the 3 Gb memory bump you see, example:
![2022-01-04170321](https://user-images.githubusercontent.com/11976836/148096582-9c35f1f9-316f-489d-9034-ab0705d2332b.jpg)

**Comment by bNobo** [comment]
Hi @filfreire,

* the problem doesn't happen anymore if I rename Insomnia folder and it comes back if I restore the folder
* I have managed to export all documents and collections just before it freezes, then renamed Insomnia folder and reimport all documents and collections. The problem does not happen anymore with the fresh folder. I have yet to see if it holds on the long term.
* I have one thing out of ordinary : I have imported request collections from Insomnia Designer, then I have converted them manually to Design Document thanks to the workaround provided by @pplotka : https://github.com/Kong/insomnia/discussions/3833?sort=old

Hope it could help.

**Comment by filfreire** [comment]
@bNobo thank you. Please feel free to open a new issue if you are still facing problems. We have a handful of [performance problems](https://github.com/Kong/insomnia/issues?q=is%3Aissue+is%3Aopen+label%3AB-performance) we'll need to look at that are in line with this one, so I'm closing it for now.
